app/models/normalize_primo_books.rb

```diff
@@ -0,0 +1,49 @@
+# Tranforms results from {SearchPrimo} into normalized {Result}s
+# for metadata that is relevant for book type results
+class NormalizePrimoBooks
+  def initialize(record)
+    @record = record
+  end
+
+  def book_metadata(result)
+    result.thumbnail = thumbnail
+    result.publisher = publisher
+    result.location = location
+    result.subjects = subjects
+    result
+  end
+
+  def thumbnail
+    return unless @record['pnx']['addata']['isbn']
```

Comment:
We may want to consider using issn on the articles normalizer to add thumbnails as well. (not asking for a change to this PR, just noting I think syndetics can also accept issn and may be worth asking UX if they want that)

Reply:
Good call. I'll make a note in the QA ticket.